server/e2e/main_test.go

```diff
@@ -0,0 +1,66 @@
+//go:build e2e
+// +build e2e
+
+package main
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/mattermost/mattermost-plugin-msteams-sync/server/msteams"
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/pelletier/go-toml/v2"
```

Comment:
nit: almost all our configuration files are `json`, and I wonder if we should stick with that for consistency?

Reply:
Done, changed to json